stable/kommander/Chart.yaml

```
@@ -9,4 +9,4 @@ maintainers:
   - name: gracedo
   - name: dkoshkin
 name: kommander
-version: 0.13.36
+version: 0.13.37
```

This includes a minor version update to kommander-ui so the chart version should also have a minor version bump.
```suggestion
version: 0.14.0
```

---

makes sense.

---

makes sense to me too. thus far kommander has (more or less) bumped patch versions for changes within a kommander release, then bumped the minor when the release goes out. I think it makes sense to actually follow semver, we just need to get everyone on the same page so it's consistent